usaspending_api/search/tests/integration/test_spending_by_award.py

```diff
@@ -981,6 +982,30 @@ def _test_correct_response_for_psc_code_object_subawards(client):
     assert resp.json().get("results") == expected_result, "PSC Code filter does not match expected result"
 
 
+def _test_eclipsed_psc_code(client):
+    resp = client.post(
+        "/api/v2/search/spending_by_award",
+        content_type="application/json",
+        data=json.dumps(
+            {
+                "filters": {
+                    "award_type_codes": ["A", "B", "C", "D"],
+                    "psc_codes": {"require": ["PSC1"], "exclude": ["PSC1"]},
```

Comment:
Alright, but what if the user requires PSC1, but excludes PSC11? Will it show results for PSC12? Will it hide results for PSC11? 

What if the user excludes PSC1, but includes PSC11? Will it do the opposite?

What if the user includes PSC1, but excludes PSC11, but includes PSC111? PSC is extra complicated, since it can go four levels deep, unlike the other two trees.